framework/include/postprocessors/InterfaceIntegralVariableValuePostprocessor.h

```diff
@@ -39,6 +39,11 @@ class InterfaceIntegralVariableValuePostprocessor : public InterfaceIntegralPost
 
   /// the type of interface value we want to compute
   const MooseEnum _interface_value_type;
+  /// the fv variable for the neighbor variable
+  const MooseVariableFV<Real> * _neighbor_fv_variable;
```

Comment:
Let's do the full deal: `const MooseVariableFV<Real> * const _neighbor_fv_variable` in case some evil developer comes along in the future and tries to change what is pointed to